src/server/server_family.cc

```diff
@@ -1012,6 +1012,7 @@ string ServerFamily::GetReplicaMasterId() const {
 
 void ServerFamily::OnClose(ConnectionContext* cntx) {
   dfly_cmd_->OnClose(cntx);
+  cntx->conn()->SetClientTrackingSwitch(false);
```

Comment:
I assume we had `MainService:: OnClose` where watch, pubsub and monitoring are cleaning up. Let's add it there so all of them are in one place